### PR TITLE
[SDK-8189] Adds Google IMA Ads BPA

### PR DESCRIPTION
**Pull request by david-almaguer**
Adds the new Google IMA Ads project for demoing how to set up a player with Google IMA.

### CODE REVIEW

File: JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            // Build the player item. This method can throw so be sure to handle the error.
+            let playerItem = try playerItembuilder.build()
+
+            // Second, use the JWImaAdvertisingConfigBuilder to create a JWAdvertisingConfig that will be used by the player configuration.
+            let adConfigBuilder = JWImaAdvertisingConfigBuilder()
+                // Set the VMAP url for the builder to use.
+                .vmapURL(vmapURL)
+
+            // Build the IMA advertising configuration. This method can throw so be sure to handle the error.
+            let adConfig = try adConfigBuilder.build()
```
Comment (s-seibert) [inline JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift:47]: To simplify, let's chain .build() after .vmapURL() to make the code more concise. This also will show the developer why we did the builder pattern. Separating it out makes it look bulkier.

File: JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        let vmapURL = URL(string: vmapUrlString)!
+
+        do {
+            // First, use the JWPlayerItemBuilder to create a JWPlayerItem that will be used by the player configuration.
+            let playerItembuilder = JWPlayerItemBuilder()
+                .file(videoUrl)
+                .posterImage(posterUrl)
+
+            // Build the player item. This method can throw so be sure to handle the error.
+            let playerItem = try playerItembuilder.build()
```
Comment (s-seibert) [inline JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift:39]: As my comment below, chain .build() after .posterImage()

File: JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            let adConfig = try adConfigBuilder.build()
+
+            // Third, create a player config with the created JWPlayerItem and JWAdvertisingConfig
+            let configBuilder = JWPlayerConfigurationBuilder()
+                .playlist([playerItem])
+                .advertising(adConfig)
+                .autostart(true)
+
+            // Build the player configuration. This method can throw so be sure to handle the error.
+            let config = try configBuilder.build()
```
Comment (s-seibert) [inline JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift:56]: Chain this too, to keep the code more concise.

**Review by s-seibert — CHANGES_REQUESTED** [review]

File: JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift (hunk 1 above)
Comment (david-almaguer) [reply JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift:47]: Addressed

File: JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift (hunk 2 above)
Comment (david-almaguer) [reply JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift:39]: Addressed

File: JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift (hunk 3 above)
Comment (david-almaguer) [reply JWBestPracticeApps/Google IMA Ads/Google IMA Ads/ViewController.swift:56]: Addressed

**Review by s-seibert — APPROVED** [review]

**Review by DavidPerezP124 — APPROVED** [review]

**Pull request merged**